src/measure_theory/measure/measure_space_def.lean

```diff
@@ -360,6 +360,9 @@ eventually_eq_empty.trans $ by simp only [ae_iff, not_not, set_of_mem_eq]
 
 @[simp] lemma ae_eq_univ : s =ᵐ[μ] (univ : set α) ↔ μ sᶜ = 0 := eventually_eq_univ
 
+lemma ae_iff' {p : α → Prop} : (∀ᵐ a ∂μ, p a) ↔ {a | p a} =ᵐ[μ] univ :=
+by rw [ae_eq_univ, compl_set_of, ae_iff]
```

Comment:
This lemma is true for any filter, not only `ae`. It is defeq `filter.eventually_eq_univ.symm`.

Reply:
OK, I've dropped this lemma.